.travis.yml

```diff
@@ -0,0 +1,21 @@
+# Borrowed from here:  https://gist.github.com/dan-blanchard/7045057
+# The validity of this file can be checked here: http://lint.travis-ci.org/
+
+language: python
+python:
+    - "2.7"
+
+# Setup anaconda
+before_install:
+  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda2/bin:$PATH
+
+# Install packages
+install:
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION --file requirements.txt
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION jupyter
+  - python setup.py install
+script: 
+  - "python setup.py test"
```

Comment:
if you want, adding an empty newline in all text/code files makes `diff`s better readable